pcdsdevices/valve.py

```diff
@@ -226,6 +226,22 @@ class VGC(VRC):
     mps_state = Cpt(EpicsSignalRO, ':MPS_FAULT_OK_RBV', kind='omitted',
                     doc=('individual valve MPS state for debugging'))
 
+class VFS(VRC):
+    """Class for Fast Shutter Valve."""
+    vfs_closed = Cpt(EpicsSignalRO, ':CLS_RBV', kind='normal', 
+                     doc='Fast Shutter Closed Status')
+    vfs_open = Cpt(EpicsSignalRO, ':OPEN_RBV', kind='normal',
+                   doc='Fast Shutter Open Status')
+    vfs_vac_fault_ok = Cpt(EpicsSignalRO, ':FAULT_OK_RBV', kind='normal',
+                           doc='Fast Shutter Vacuum Fault OK')
+    vfs_vac_fault_reset = Cpt(EpicsSignalWithRBV, ':FAULT_RESET', kind='normal'
+                              , doc='Fast Shutter Vacuum Fault Reset'
```

Comment:
Neither way is pretty but I think this is better. Also missing a closing paren

```suggestion
    vfs_vac_fault_reset = Cpt(EpicsSignalWithRBV, ':FAULT_RESET',
                              kind='normal',
                              doc='Fast Shutter Vacuum Fault Reset')
```